nemo/collections/asr/losses/pt_losses/contrastive.py

```diff
@@ -48,12 +48,13 @@ def __init__(
         proj_dim: int = 128,
         combine_time_steps: int = 1,
         num_negatives: int = 100,
-        quantized_targets: bool = True,
+        quantized_targets: bool = False,
```

Comment:
It is fine for now since we have not released pretrained checkpoints, however changing default like this will break older Nemo files. Consider that when designing / adding breaking changes